src/services/me/index.js

```diff
@@ -33,9 +33,14 @@ class Service {
 			throw new Forbidden('Your access token is not valid.');
 		}
 		user.accountId = params.account._id;
+
+		const roles = await this.app.service('/roles/user').get(userId, params);
```

Comment:
They go over the user services with populate roles and the users services resolve all permissions if the roles are populated.